openedx/features/enterprise_support/enrollments/tests/test_utils.py

```diff
@@ -160,16 +157,15 @@ def test_existing_enrollment_does_not_fail(
     ):
 
         expected_response = None
-        enrollment_response = {'mode': COURSE_MODE, 'is_active': True}
 
         mock_add_enrollment_api.side_effect = CourseEnrollmentExistsError("test", {})
         mock_tx.return_value.atomic.side_effect = None
 
-        mock_get_enrollment_api.return_value = enrollment_response
+        mock_get_enrollment_api.return_value = expected_response
 
         mock_user_model.return_value = self.a_user
 
-        response = lms_enroll_user_in_course(USERNAME, COURSE_ID, COURSE_MODE, ENTERPRISE_UUID)
+        response = lms_update_or_create_enrollment(USERNAME, COURSE_ID, COURSE_MODE, True, ENTERPRISE_UUID)
```

Comment:
nit: perhaps use the kwarg property to be more readable at first glance, e.g. `is_active=True`